build.gradle

```diff
@@ -71,6 +71,7 @@ dependencies {
     implementation "dev.langchain4j:langchain4j-milvus"
     implementation "dev.langchain4j:langchain4j-pinecone"
     implementation "dev.langchain4j:langchain4j-pgvector"
+    implementation "dev.langchain4j:langchain4j-community-redis:1.2.0-beta8"
```

Comment:
You may use Langchain4J 1.2.0 instead of hardcoding this version only.
It also creates inconsistencies as it's a 1.2 but we're using 1.0